.Lib9c.Tests/TableSheets.cs

```diff
@@ -1,133 +1,124 @@
-using System;
-using System.Text;
-using Libplanet.Action;
-using LruCacheNet;
-using Nekoyume.Model.State;
-
-namespace Nekoyume.TableData
+namespace Lib9c.Tests
```

Comment:
is it okay we move this exposed class to the test project?

Reply:
I think it was a test fixture. It should be prevented from being called out by external projects.